lib/tasks/import_markets.rake

```diff
@@ -0,0 +1,13 @@
+require './db/csv/farmers_market_generator.rb'
```

Comment:
Think you'll need to change this to `model_generator` because this task calls on that instead of farmers market generator

Reply:
Updated